Add note for Hermes bundle manually (#5231)
getsentry/sentry-docs: src/platforms/react-native/manual-setup/hermes.mdx

### Compile Sourcemaps

<Note>

Currently, there is an issue when bundling the `hermes` bundle and `source maps` manually.
We recommend not bundling these manually until this is fixed.

> Follow the [issue](https://github.com/facebook/react-native/issues/34043) for more details.

Author: @imatwawana Do you know if linking issues work within a `Note`? Using the [ ] ( ) format, it's not working in the preview.

Reviewer: Adding: as long as you leave a line break between the text of the note and the opening and closing Note tags, markdown formatting should always work.

Author: Oh that was the typo, thanks.

> We're currently facing issues when bundling the `hermes` bundle and `source maps` manually.
> We'd recommend not bundling it manually for now till this issue gets fixed.
> Follow the [issue[(https://github.com/facebook/react-native/issues/34043) for more details.

Reviewer: ```suggestion
Currently, there is an issue when bundling the `hermes` bundle and `source maps` manually.
We recommend not bundling these manually until this is fixed.
Follow the [issue](https://github.com/facebook/react-native/issues/34043) for more details.
```
